Formula/autoenv.rb

```diff
@@ -25,3 +25,17 @@ def caveats
     assert_match "it works", shell_output(testcmd)
   end
 end
+__END__
```

Comment:
```suggestion

__END__
```
We usually leave an empty line here. (I don't really know why, but I remember reading this somewhere in the docs and all the other formulae do it..)